.github/workflows/docker.yaml

```diff
@@ -3,12 +3,6 @@ name: docker
 on:
   schedule:
     - cron: '0 18 * * *'
-  push:
-    branches:
-      - 'v[0-9]+.*'
```

Comment:
Why remove this ?

Reply:
Every PR is already required to be tested based on the upstream. At the meantime, it seems that no one cares about the status of the push event, at present.

Of course, we could add it back if necessary in future.

Reply:
Emmm. Need add it again when need release.

Reply:
> Emmm. Need add it again when need release.

This is an awesome idea. We should conceive of a workflow about releasing, including CI, FO and benchmarks, etc.

Reply:
I'm gonna mark it down and merge this PR.

Reply:
this has been covered by release workflow.